recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -117,7 +117,7 @@ def _configure_cmake(self):
         if tools.Version(self.version) <= "2.1.0":
             self._cmake.definitions["CMAKE_MACOSX_BUNDLE"] = False # avoid configuration error if building for iOS/tvOS/watchOS
 
-        if tools.cross_building(self):
+        if tools.cross_building(self) and not self.settings.os == "Android":
```

Comment:
This is not an actual fix.
The real bug is conan not respecting `CONAN_CMAKE_SYSTEM_PROCESSOR` when cross building.

See this bug report: https://github.com/conan-io/conan/issues/9736

Reply:
Does conan automagically set `CONAN_CMAKE_SYSTEM_PROCESSOR`?

Reply:
Yes, to `self.settings.arch`.
Overriding it in the profiles does not work when cross building.
You can find a short repro in the issue (without requiring downloading the android ndk).

Reply:
> Yes, to self.settings.arch.

But it's wrong anyway in some cases.

Reply:
Indeed, my issue is about being unable to override `CONAN_CMAKE_SYSTEM_PROCESSOR`.